lib/core.js

```diff
@@ -207,6 +207,11 @@ declare class RegExp {
 }
 
 declare class Date {
+    // new Date();
```

Comment:
Hi Jan - if you could add a comment here noting that we want to use constructor overloads once they become available, that would be great.